src/main/java/br/com/diegomelo/cleanparse/core/Parser.java

```diff
@@ -34,11 +33,17 @@
 	 * @throws ParserException
 	 */
 	protected void validate(Object obj) throws ParserException {
+String password = "123456";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'password' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://35.202.0.216/coding_rules#rule_key=squid%3AS2068)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "password". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://35.202.0.216/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "password" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://35.202.0.216/coding_rules#rule_key=squid%3AS1481)